### PR TITLE
Add RCOutputRGBLedInverted

### PR DESCRIPTION
**Pull request by staroselskii**
Some RGB leds controlled with an inverted logic. This class addresses
this.

### CODE REVIEW

File: libraries/AP_Notify/RCOutputRGBLed.h [hunk 1]
```diff
@@ -13,13 +13,24 @@ class RCOutputRGBLed: public RGBLed {
 protected:
     bool hw_init() override;
     virtual bool hw_set_rgb(uint8_t red, uint8_t green, uint8_t blue) override;
+    virtual uint16_t get_duty_cycle_for_color(const uint8_t color, const uint16_t usec_period);
```
Comment (lucasdemarchi) [inline libraries/AP_Notify/RCOutputRGBLed.h]: missing const at the end (which is more important than the const in the args

File: libraries/AP_Notify/RCOutputRGBLed.h [hunk 2]
```diff
@@ -13,13 +13,24 @@ class RCOutputRGBLed: public RGBLed {
...
 private:
     uint8_t _red_channel;
     uint8_t _green_channel;
     uint8_t _blue_channel;
 };
 
+class RCOutputRGBLedInverted : public RCOutputRGBLed {
+public:
+    RCOutputRGBLedInverted(uint8_t red_channel, uint8_t green_channel,
+                      uint8_t blue_channel)
```
Comment (lucasdemarchi) [inline libraries/AP_Notify/RCOutputRGBLed.h]: wrong alignment

**Comment by lucasdemarchi** [comment]
Some nitpicks... otherwise LGTM

**Comment by staroselskii** [comment]
@lucasdemarchi 

I forced pushed the requested changes. Hope, everything's fine now.

**Pull request merged**